coalib/bears/GlobalBear.py

```diff
@@ -4,22 +4,28 @@
 
 class GlobalBear(Bear):
     """
-    A GlobalBear is able to analyze semantic facts across several file.
+    A GlobalBear analyzes semantic facts across several files.
 
     The results of a GlobalBear will be presented grouped by the origin Bear.
-    Therefore Results spanning above multiple files are allowed and will be
-    handled right.
+    Therefore Results spanning across multiple files are allowed and will be
+    handled correctly.
 
-    If you only look at one file at once anyway a LocalBear is better for your
-    needs. (And better for performance and usability for both user and
-    developer.)
+    If you are inspecting a single file at a time, you should consider
+    using a LocalBear.
```

Comment:
No, we should not. Performance --- there are no measurements. Usability is too abstract. It is just architecture: `GlobalBear` is for several files, `LocalBear` is just for one. That point is clear.